lib/addons/p5.dom.js

```diff
@@ -675,6 +675,84 @@
   };
 
   /**
+   * Translates an element relative to (0, 0, 0) of the window through
+   * css transforms. 
+   * @method  translate
+   * @param  {Number} x x-position relative to upper left of window
+   * @param  {Number} y y-position relative to upper left of window
+   * @param  {Number} z z-position
+   * @return {p5.Element}
+   * @example
+   * <div><code class='norender'>
+   * function setup() {
+   *   var cnv = createCanvas(100,100);
+   *   //translates canvas 50px to the right and 100px
+   *   //below upper left corner of the window
+   *   cnv.translate(50,100);
+   * }
+   * </code></div>
+   */
+  p5.Element.prototype.translate = function(x, y, z){
+    this.elt.style.position = 'absolute';
+    var regEx = new RegExp('translate');
+    var substr = '';
+    while (regEx.test(this.elt.style.transform)){
+      var min = this.elt.style.transform.indexOf('translate');
+      var max = this.elt.style.transform.indexOf(')');
+      if (min != -1) {
+        substr = this.elt.style.transform.substring(min, max+1);
+        this.elt.style.transform = this.elt.style.transform.replace(substr, '');
+      }
+    }
+    this.elt.style.transform += 'translate3d('+x+'px,'+y+'px,'+z+'px)';
```

Comment:
it would probably be good to check here whether there are 2 or 3 arguments, and use regular translate rather than translate3d if only x and y are provided.

style wise, when there is a variable number of arguments, we generally leave the params empty and use `arguments.length` to test how many there and `arguments[0]` etc to grab them.